rest/client/annotated_wrapper.go

```diff
@@ -0,0 +1,27 @@
+package client
+
+import (
+	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/message"
+	"github.com/mongodb/grip/send"
+)
+
+type taskAnnotator struct {
+	send.Sender
+	taskID  string
+	logType string
+}
+
+func (l *taskAnnotator) Send(m message.Composer) {
+	grip.Error(m.Annotate("task_id", l.taskID))
+	grip.Error(m.Annotate("type", l.logType))
```

Comment:
make the keys constants? 

should we make this genericish and put it in grip itself?